public/html/fragments/blog-list.html

```diff
@@ -22,20 +22,23 @@ <h3>
     </article>
     <article class="blog-post">
         <h3>
-            <a href="/blog/post/ai-decision-making">The Decision Revolution: Leveraging AI for Superior Decision Making</a>
+            <a href="/blog/post/importance-of-prds-for-ai">The Importance of PRDs for AI Work: Guiding Your Digital Workforce</a>
```

Comment:
These fragments don't exist anymore, please rebase so you are off of main. These changes won't be needed at all. So please make sure this is undone safely. I recommend removing the change from here, as if it never happened.